src/app/components/modules/simplified-lunch/simplified-lunch.component.scss

```diff
@@ -0,0 +1,269 @@
+@import '../../../../styles/settings';
+
+/* --------------------------
+  Main Styles
+--------------------------- */
+.module-container{
+  width:100%;
+  height:100%;
+}
+.module-container[max-height~='500px']{
+  .not-main-entree{
+    display: none;
+  }
+  .lunch-category{
+    border-top: none;
+  }
+}
+.module-container[max-width~='500px']{
+  .not-main-entree{
+    display: none; 
+  }
+  .lunch-category{
+    border-top: none;
+  }
+}
+.module-container[max-height~='300px']{
+  .lunch-date{
+    display:none;
+  }
+  .lunch-category{
+    padding-top: 0rem;
+  }
+
+}
+
+.lunch-week .lunch-category{
+  border-top: 1px solid rgba(white, 0.5);
+
+}
+.loading-container {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+
+	.loading {
+		width: 35rem;
+		padding: 1rem;
+		text-align: center;
+		color: $sophisticated-white;
+
+		h1 {
+			font-size: 3rem;
+			font-weight: 300;
+		}
+	}
+}
+
+.lunch-container {
+	padding: 2rem 0 0 0;
+	margin-bottom: 6rem;
+	.school-selection {
+		margin-bottom: 1rem;
+		text-align: center;
+
+		.btn {
+			font-size: small;
+		}
+	}
+
+	.week-navigation {
```

Comment:
There's no week navigation in this lunch module. You should delete these and any other unused styles copied over from the lunch page.